core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changelog.xml

```diff
@@ -77,4 +77,5 @@
     <include file="changesets/20201105_add_use_reference_disks.xml" relativeToChangelogFile="true"/>
     <include file="changesets/20210111_retroactive_default_cromwell_backend_per_project.xml" relativeToChangelogFile="true"/>
     <include file="changesets/20210129_add_entity_statistics_cache_tables.xml" relativeToChangelogFile="true"/>
+    <include file="changesets/20210407_add_entity_attr_temp_table_procedure.xml" relativeToChangelogFile="true"/>
```

Comment:
for this PR or a future one, let's drop the existing `createTempTables` stored proc

Reply:
Filed as AS-722